src/modules/Crowbar.rb

```diff
@@ -118,6 +93,13 @@ def json2hash(file_name)
       ret
     end
 
+    # read given yaml file and return the content as a map
+    def yaml2hash(file_name)
+      ret = YAML.load(File.read(file_name))
```

Comment:
It doesn't properly update the widget with Repository URL. When you go through table, once you meet some repository where URL is set, this one is shown down in Repo URL widget also for other repos.

Just add
`url = "" if url.nil?`
to InitRepoURL, before the ChangeWidget call, and it fixes this issue.